src/applications/personalization/profile/components/connected-apps/AppDeletedAlert.jsx

```diff
@@ -5,13 +5,37 @@ import { focusElement } from 'platform/utilities/ui';
 
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 
-export const AppDeletedAlert = props => {
+export const AppDeletedAlert = ({
+  dismissAlert,
+  id,
+  privacyPolicies,
+  title,
+}) => {
   useEffect(() => {
     focusElement('[data-focus-target]');
   }, []);
-
-  const { id, title, dismissAlert } = props;
-  const alertMessage = `We’ve disconnected ${title}. This app can’t access any new information from your VA.gov profile. Some apps may still store information you’ve already shared. If you’d like to ask the app to delete any stored information, contact the app’s support.`;
+  const privacyLink = (
+    <a
+      href={`${privacyPolicies[title]}`}
```

Comment:
It seems like it's possible for this `href` to be undefined if the call to look up the privacy policy URL is still pending or failed.

Reply:
Will be resolved by updates to the `GET profile/connected_applications` response.